.gitignore

```diff
@@ -12,3 +12,4 @@ bundles/
 data
 opactl_*
 files
+.go
```

Comment:
Hm? Where would that file come from? 🤔

Reply:
https://github.com/open-policy-agent/opa-control-plane/pull/74/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R145